stl/inc/xutility

```diff
@@ -5573,7 +5579,15 @@ _INLINE_VAR constexpr bool _Vector_alg_in_find_is_safe = // Can we activate the
 #endif // __cpp_lib_byte
         conjunction<is_integral<_Ty>, is_integral<_Elem>>, // We're finding an integer in a range of integers.
         // The integer types can be different, which requires careful handling.
-        conjunction<is_pointer<_Ty>, is_same<_Ty, _Elem>>>; // We're finding a U* in a range of U* (identical types).
+        conjunction<is_pointer<_Ty>, is_same<_Ty, _Elem>>,
+        // We're finding a pointer in a range of pointers of the same type (U* or U(*)(Ts...)).

```

Comment:
it's confusing to have these comments below what they apply to, we could at least put `^`s in